### PR TITLE
Remove litecoin support

### PR DESCRIPTION
**Pull request by SvenSchnieders**
## Description:
Deleted all Lightcoin related code.

## Motivation and Context:

This finally ends the rumor that Zap "supports" Litecoin.

## How Has This Been Tested?
The code has not been tested. I'm new to this but I hope this still helps somehow.

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have reviewed and updated the documentation accordingly.
- [x] I have read the _CONTRIBUTING_ document.
- [ ] I have added tests to cover my changes where needed.
- [ ] All new and existing tests passed.
- [ ] My commits have been squashed into a concise set of changes.

### CODE REVIEW

File: config/custom-environment-variables.json [hunk 1]
```diff
@@ -60,23 +60,12 @@
...
-        "mainnet": {
-          "__name": "NEUTRINO_LITECOIN_MAINNET",
-          "__format": "json"
-        }
       }
     }
   },
 
   "units": {
     "bitcoin": "UNITS_BITCOIN",
```
Comment (mrfelton) [inline config/custom-environment-variables.json]: This is invalid json due to the trailing comma and as a result the app will not start with this change.

Comment (SvenSchnieders) [reply config/custom-environment-variables.json]: Ithink I fixed it

Comment (SvenSchnieders) [reply config/custom-environment-variables.json]: Thank you, I think I fixed it.

**Comment by bolatovumar** [comment]
@mrfelton is PR abandoned? I can continue this work if we want it continued.

**Comment by mrfelton** [comment]
We'd still like to action this @bolatovumar . I'm not sure why OP closed the issue although the original PR needs quite a bit of work to make it functional as it broke many things in the process.

**Comment by bolatovumar** [comment]
@mrfelton ok, I can work on this slowly and get this done eventually.

**Comment by SvenSchnieders** [comment]
I closed it because I wasn't able to make it work. Also wasn't sure if improving my PR wouldn't be more work for you guys than starting from "scratch."

**Comment by mrfelton** [comment]
This has now been addressed via https://github.com/LN-Zap/zap-desktop/pull/3264.

Thanks for the effort @SvenSchnieders .

**Pull request closed without merging**